_component_design/toast.md

```diff
@@ -5,40 +5,136 @@ layout: component
+
+Clicking on the “X” closes the toast, but it will go away after approx. 10 seconds if no action is taken. (See “Timing” section below for details.) 
+
+While height of toasts are dependent on the amount of text, the minimum height is 56px without a CTA. If a CTA or more lines are needed, the height can increase, but try to keep notification messages short and succinct, and up to 4 lines max. Use complete sentences as it leaves less room for ambiguity. 
+
+Icons are always centered vertically. 
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Default toast notifications specifications"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/notifications/toasts/toast-specifications.svg)
 {% endfigure %}
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Action"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/toast-alert-action.svg){:width="100%"}
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Timing
+
+The toast fades into place over 400ms, sits in place for eight seconds, and fades out over 400ms. If the user happens to hover over the toast while it is fading out, it reanimates immediately (0.0ms) back to full opacity. Likewise, if the user is hovering over the toast at any point, it persists. When the mouse is removed from the expired toast, the toast persists for three more seconds then fades over the normal 400ms. 
+
+{% endcolumn %}
+
+</div>
+
+## Variations
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Without a CTA (call to action)
+
+Clicking on the “X” dismisses the toast notification. There are no other actions that can be taken on this type of toast notification.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Toast notifications without CTA (call to action)"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/notifications/toasts/toast-wo-cta.svg)
 {% endfigure %}
 {% endcolumn %}
+
 </div>
+
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Action Buttons"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/toast-alert-action-buttons.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### With a CTA
+
+There should only be one CTA per toast.
+
+In the Informational toast notification example here, the height of the toast increases to 96px to maintain the 8px spacing between the text and the CTA, and the 12px spacing between the CTA and the bottom of the toast. (The width always stays consistent at 350p
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Toast notifications with CTA (call to action)"] [class:"image bg-light border"] %}
```

Comment:
s/`Toast notifications with CTA (call to action)`/`Toast notifications with CTA`

Reply:
updated.